File: src/Reference.CSharp/OpenStreetMapItem.cs
using Newtonsoft.Json;

namespace Reference.CSharp
{
    /// <summary>
    /// OpenStreet Map
    /// </summary>
    public class OpenStreetMapItem
    {
        /// <summary>
        /// Place Id
        /// </summary>
        [JsonProperty("place_id")]
        public long PlaceId { get; set; }
        /// <summary>
        /// Display name
        /// </summary>
        [JsonProperty("display_name")]
        public string DisplayName { get; set; }
        /// <summary>
        /// Address
        /// </summary>
        [JsonProperty("address")]
        public OpenStreetMapAddress Address { get; set; }
    }
}

File: src/Reference.VB/OpenStreetMapItem.vb
Imports Newtonsoft.Json

''' <summary>
''' OpenStreet Map
''' </summary>
Public Class OpenStreetMapItem
    ''' <summary>
    ''' Place Id
    ''' </summary>
    <JsonProperty("place_id")>
    Public Property PlaceId As Long
    ''' <summary>
    ''' Display name
    ''' </summary>
    <JsonProperty("display_name")>
    Public Property DisplayName As String
    ''' <summary>
    ''' Address
    ''' </summary>
    <JsonProperty("address")>
    Public Property Address As OpenStreetMapAddress
End Class


